pushy/src/main/java/com/turo/pushy/apns/util/ApnsPayloadBuilder.java

```diff
@@ -530,6 +535,58 @@ public ApnsPayloadBuilder setThreadId(final String threadId) {
         return this;
     }
 
+    /**
+     * <p>Sets the summary argument for this notification. The summary argument is:</p>
+     *
+     * <blockquote>The string the notification adds to the category’s summary format string.</blockquote>
+     *
+     * <p>For example when defining an UNNotificationCategory, passing a format string like
```

Comment:
It's implied, but I might explicitly call out that the "summary format string" mentioned in the quote above is defined by the receiving app (as opposed to whoever is sending the push notification). Not all users of Pushy will be familiar with the details of the iOS SDK that actually handles notifications; with that in mind, taking some extra caution to explain when things are outside of Pushy's scope can be a big help.

Also, as a suuuuuper nitpick, could we do `{@code UNNotificationCategory}`? That will format the text as monospace and make it look more, y'know, code-y.

Reply:
> mentioned in the quote above is defined by the receiving app

Makes sense. Done in cd6eb42. Let me know if you'd like to reword.